src/components/AvatarBadge/AvatarBadge.style.ts

```diff
@@ -0,0 +1,72 @@
+import { Avatar, Badge, Avatar as MuiAvatar } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import type { BadgeOffsetProps } from './AvatarBadge';
+import { getAvatarMask } from './getAvatarMask';
+
+interface StyledAvatarProps {
+  avatarSize: number;
+  badgeSize: number;
+  badgeOffset?: BadgeOffsetProps;
+  badgeGap?: number;
+}
+
+// Styled Avatar component for the badge
+export const StyledAvatar = styled(Avatar, {
+  shouldForwardProp: (prop) =>
```

Comment:
You do not use mui at any point into the image generation, would be easier and cleaner to just add some styling yourself there

Reply:
I added this component and it will be used later on. In another PR I will make use of that.

Reply:
But mui is not used at all in there, I would rather prefer to: 
Or use it instead of every div
Or not use it at all

Reply:
When I first started to generate images via NextJS I was doing it on the client-side ad MUI worked well - that´s also when I added this component for "BadgeAvatars" which can be used in various places and comes with easier usage.

Using this similar approach I created a AvatarBadgeSSR component, similar to the client component which is now being used for the image-generation.

To not blow up this PR with more tasks unrelated  to the actual ticket - I have not used the MUI parts, yet. But am planning. I could remove those client-side MUI changes here and add a seperate PR instead.